Cargo.toml

```diff
@@ -26,6 +26,7 @@ travis-ci = { repository = "amethyst/laminar", branch = "master" }
 byteorder = "1.2"
 crc = "1.8"
 crossbeam-channel = "0.3"
+either = "1.5.2"
```

Comment:
Must we add this? I really dislike adding small dependencies that can be written here in less than 15 mins

Reply:
I honestly think Either should be part of std, I don't like small dependencies either but everyone's familiar with Either, hence why. Can easily be replaced by a little enum though.

Reply:
Please do :)